dev/lint-python

```diff
@@ -61,13 +88,28 @@ else
 fi
 
 if [ "$lint_status" -ne 0 ]; then
-    echo "Python lint checks failed."
-    cat "$PYTHON_LINT_REPORT_PATH"
+    echo "PEP8 checks failed."
+    cat "$PEP8_REPORT_PATH"
+else
+    echo "PEP8 checks passed."
+fi
+
+rm "$PEP8_REPORT_PATH"
+
+
+for to_be_checked in "$PATHS_TO_CHECK"
+do
+    pylint --rcfile="$SPARK_ROOT_DIR/pylintrc" $to_be_checked >> "$PYLINT_REPORT_PATH"
```

Comment:
I am not completely sure of this. There are certain warnings that are useful also which are not raised by the `--errors-only` flag.